source/extensions/common/wasm/context.cc

```diff
@@ -1589,7 +1589,14 @@ void Context::setEncoderFilterCallbacks(Envoy::Http::StreamEncoderFilterCallback
   encoder_callbacks_ = &callbacks;
 }
 
-void Context::onHttpCallSuccess(uint32_t token, Envoy::Http::ResponseMessagePtr& response) {
+void Context::onHttpCallSuccess(uint32_t token, Envoy::Http::ResponseMessagePtr&& response) {
+  if (proxy_wasm::current_context_ != nullptr) {
```

Comment:
Could you turn this into a function? Accessing fields across library boundaries isn't the best practice, IMHO.

Reply:
I'll add a TODO for that.   It requires changing the -host code which is a two step dance.  I have a bunch of queued things to do there and I'll group this in.